From f58ae13c60b8ed9168fc97e1886a505030e968c5 Mon Sep 17 00:00:00 2001
From: Zhang Ying-22455 <ying.zhang22455@nxp.com>
Date: Fri, 25 May 2018 11:29:15 +0800
Subject: [PATCH 516/767] arm64: dts: ls1088a: Add the identify of the platform
 to support to set rcpm bit

Add the identify of the platform to support set the rcpm

Signed-off-by: Zhang Ying-22455 <ying.zhang22455@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
index cb74343436be..f01260363d0c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -625,8 +625,10 @@
 		};
 
 		ftm0: ftm0@2800000 {
-			compatible = "fsl,ftm-alarm";
-			reg = <0x0 0x2800000 0x0 0x10000>;
+			compatible = "fsl,ls1088a-ftm-alarm";
+			reg = <0x0 0x2800000 0x0 0x10000>,
+			      <0x0 0x1e34050 0x0 0x4>;
+			reg-names = "ftm", "pmctrl";
 			interrupts = <0 44 4>;
 		};
 
-- 
2.17.0

